.travis.yml

```diff
@@ -27,18 +28,16 @@ install:
   # Useful for debugging any issues with conda
   - conda info -a
   # Create conda test environment
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy pandas xarray requests beautifulsoup4 lxml netCDF4 pytest-cov pytest-ordering future
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy pandas xarray requests dask toolz netCDF4 portalocker pytest-cov pytest-ordering
   - conda activate test-environment
   # Get latest coveralls from pip, not conda
-  - pip install coveralls
-  - pip install pytest-flake8
+  - pip install coveralls pytest-flake8
```

Comment:
We could potentially use test_requirements.txt as in the core code.  See pysat/pysat#600

Reply:
You know I like that.